runcoms/README.md

```diff
@@ -10,61 +10,199 @@ in */etc* by default.
+in `zshrc` or, in `${ZDOTDIR:-$HOME}/.zlogin` . Note however that the
+`zlogin` (sysem or user) files will be sourced **only for login shells**. 
+
+That means, when logging in to a remote system via ssh for example or 
+when calling say, `XTerm` or your favorite emulator by specifying you 
+want a **login** shell (by calling `zsh -ls` instead of plain `zsh`). 
+Otherwise, the `zlogin` files are ignored. 
+
+Also, note that another difference between interactive login shells and 
+plain interactive shells, is that, besides sourcing `zlogin` files, 
+interactive login shells also allocate a pseudo-tty (you can see them when
+invoking `w`. Login shells have a pty assigned, while interactive non-login
+shells don't.)
+
+Why is this designed this way? because login shells are assumed to be 
+the main point of interaction with a user  while non login shells would 
+be spawned as _sub-shells_ of a *login shell* , when executing scripts 
+that call `zsh` as a command interpreter, for example, when the *shebang* is 
+`#!/usr/bin/env zsh` or equivalent. 
+
+When launching a new terminal within an *X11* session, it is safe to 
+assume that you are *already logged in* so any terminal emulator will 
+launch an *interactive-non-login* shell. Which is, _per specification_ 
+the correct behavior, but not the behavior most users *expect*.
+
+
+There is a reason for this _specified_ behaviour:  `zlogin` files should 
+be used for *interaction oriented settings* such as custom completion, 
+keybindings, `tmux` session setup, and most `alias`es.
+
+The advantage of grouping all interaction oriented settings in 
+`.zlogin` is that non-interactive shells (executing scripts via 
+`zsh -c [script]`, the *shebang*, subshells or `make` invoked shells, 
+for example) will not be burdened by additional config settings making 
+them faster and more responsive, as _keybindings_ and _aliases_ are not 
+even loaded at startup, after all, an _executable non-interactive script_, 
+should never call on keybindings or aliases (assuming they were  properly
+written to be portable)
+
+
+Finally, it is a common source of frustration amongst graphical environment 
+users, to spend some time adding their customizations to `.zlogin` only 
+to see them ignored when launching a new `XTerm` or `tmux` pane. 
+
+This can be solved by calling (or aliasing) your terminal emulator with 
+the required option to invoke a *login* shell, for `XTerm`, for example 
+you would call `xterm -e zsh -ls` on tmux you may add one of the following 
+lines to your `.tmux.conf`:
+
+```config
+#tmux.conf:
+
+        set -g default-command 'exec /usr/bin/zsh -ls' #this will make zsh a login shell AND a session leader
+        set -g default-shell '/usr/bin/zsh -ls'
+```
+
+Here is an examplefor *i3wm* on ubuntu
+```bash
+
+#i3wm.conf:
+        ...
+        set $XTERM_CMD='xterm -e zsh -ls`
+        bind $mod+x $XTERM_CMD
+        ...
+```
```

Comment:
This is a nit-pick, but I prefer to see new-lines between paragraphs, even if it's just a code block.

Reply:
ind the end i decided to drop the indent and make the example more concise. I purposely added something that looked like ellipsis ( . . .  ) to mean that we were only looking at a section of thee file and that further lines followed, as well as more lines before...

maybe using ellipsis is not as common a convention as i thought it was... :thinking: